lib/txt/news

```diff
@@ -1,3 +1,5 @@
+01-29-18 : Jammed locks don't affect doorbash.
```

Comment:
The commit message about flagging the exit doesn't do justice to all the nice things happening here!